### PR TITLE
Removed venue bug from sessions page

### LINKED ISSUE #1658: Incorrect link of venue in sessions page
**Actual Behaviour**
Currently on the sessions page if a venue is clicked the page redirects to the rooms page but the page does not scroll to the correct venue.

**Expected Behaviour**
When venue is clicked inside the sessions page the page should redirect to rooms page, scrolling to the clicked section.

**Steps to reproduce it**
- Go to https://sched.eventyay.com/MozillaAllHands2017/sessions/session_1001.html
- Click on the venue `Yosemite Room`

### PR DESCRIPTION
**Pull request by agbilotia1998**
Fixes #1658 

Links for the change: 
https://agbilotia1998.github.io/MozillaSample

### CODE REVIEW

**Comment by agbilotia1998** [comment]
@geekyd please review!

**Comment by agbilotia1998** [comment]
@geekyd please review!

**Comment by agbilotia1998** [comment]
@geekyd done please review https://agbilotia1998.github.io/MozSample/

File: src/backend/generator.js [hunk 1]
```diff
@@ -370,7 +370,7 @@ exports.createDistDir = function(req, socket, callback) {
                   sessionObj.font_color = trackArr[i].font_color;
                   sessionObj.track_title = trackArr[i].title;
                   sessionObj.track_jump_link = '../tracks.html#' + trackArr[i].slug;
-                  sessionObj.room_jump_link = '../rooms.html#' + 'venue-' + sessionObj.startDate + '-' + fold.replaceSpaceWithUnderscore(sessionObj.location);
+                  sessionObj.room_jump_link = '../rooms.html#' +  sessionObj.location;
```
Comment (geekyd) [inline src/backend/generator.js]: The format should be `date-venue`

File: src/backend/templates/rooms.hbs [hunk 2]
```diff
@@ -98,7 +98,7 @@
                     </div>
                   </div>
                   {{#venue}}
-                    <div class="venue-filter row">
+                    <div class="venue-filter row" id="{{venue}}">
```
Comment (geekyd) [inline src/backend/templates/rooms.hbs]: The format should be `date-venue`

**Comment by geekyd** [comment]
@agbilotia1998 In the current scenario, the page scrolls to the same position regardless of the date of the particular session. We need to add date to the id.

**Comment by agbilotia1998** [comment]
Okay @geekyd I will make the changes.

**Comment by agbilotia1998** [comment]
@geekyd done please review https://agbilotia1998.github.io/SampleMozilla/

**Comment by agbilotia1998** [comment]
@geekyd done please review.

**Comment by agbilotia1998** [comment]
@geekyd done please review.

**Comment by agbilotia1998** [comment]
@geekyd done! please review.

**Comment by agbilotia1998** [comment]
@geekyd done! please review.

**Comment by geekyd** [comment]
@agbilotia1998 Fix codacy issues.

**Comment by agbilotia1998** [comment]
@geekyd done please check.

**Pull request merged**